__tests__/unit/fetch-client.test.ts

```diff
@@ -137,10 +138,25 @@ describe("fetch client", () => {
     } catch (e) {
       if (e instanceof NetworkError) {
         expect(e.message).toEqual(
-          "The network connection encountered a problem."
+          "The network connection encountered a problem.",
         );
         expect(e.cause).toBeDefined();
       }
     }
   });
+
+  it("uses the path provided in the HttpRequest if provided", async () => {
+    expect.assertions(1);
+    fetchMock.mockResponseOnce(JSON.stringify({}), {
+      headers: { "content-type": "application/json" },
+    });
+    await fetchClient.request({
+      ...dummyRequest,
+      path: "/non-the-default-api" as SupportedFaunaAPIs,
+    });
+    expect(fetchMock).toHaveBeenCalledWith(
+      expect.stringContaining("/non-the-default-api"),
```

Comment:
the previous tests implicitly set the path to "/query/1", but none of them assert on it.

can you add an assertion so the default path case is tested?